### PR TITLE
feat(HTTP): Integrate AddRowDouble and update tests

### PR DESCRIPTION
**Pull request by izikdepth**
## Description

Briefly describe the changes introduced by this pull request.

## Related issue(s)

If this Pull Request is related to an issue, mention it here.
- Fixes #(issue number)

### CODE REVIEW

File: www/http/blockchain_test.go [hunk 1]
```diff
@@ -205,6 +205,17 @@ func TestValidator(t *testing.T) {
...
 	})
+
+	t.Run("Shall return a validator", func(t *testing.T) {
+		w := httptest.NewRecorder()
+		r := new(http.Request)
+		r = mux.SetURLVars(r, map[string]string{"address": val.Address().String()})
+		td.httpServer.GetValidatorHandler(w, r)
+
+		assert.Equal(t, w.Code, 200)
+		assert.Contains(t, w.Body.String(), "0.987")
```
Comment (kehiy) [inline www/http/blockchain_test.go:216]: you can move this line to 167.

**Review by kehiy — CHANGES_REQUESTED** [review]

File: www/http/blockchain_test.go (hunk 1 above)
Comment (kehiy) [reply www/http/blockchain_test.go:216]: under the first part of test.

Comment (izikdepth) [reply www/http/blockchain_test.go:221]: done

Comment (b00f) [reply www/http/blockchain_test.go:221]: It looks to me duplicated!

Comment (izikdepth) [reply www/http/blockchain_test.go:207]: > It looks to me duplicated!

fixed.

**Review by kehiy — APPROVED** [review]

**Pull request merged**